cypress/e2e/specialFeatures/atiAnalytics/assertions/recommendations.js

```diff
@@ -42,10 +42,7 @@ export const assertRecommendationsComponentClick = ({
       });
 
       // Click on last item
-      cy.get('[data-e2e="recommendations-heading"]')
-        .find('a')
-        .last()
-        .click({ force: true });
+      cy.get('[data-e2e="recommendations-heading"]').find('a').last().click();
```

Comment:
I am wondering if we should revert this change, given that it's not related to the audio / podcast page tests?

Reply:
Let's not forget to remove this before we merge